save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -143,10 +149,16 @@ class WarnPlugin(
         // NOTE: SAVE will pass relative paths of Tests (calculated from testRootConfig dir) into the executed tool
         val fileNamesForExecCmd =
                 warnPluginConfig.wildCardInDirectoryMode?.let {
-                    // a hack to put only the directory path to the execution command
+                    // a hack to put only the root directory path to the execution command
                     // only in case a directory mode is enabled
-                    "${copyPaths[0].parent!!}$it"
+                    var testRootPath = testConfig.directory.parent ?: testConfig.directory
+                    while (testRootPath.parent != null && testRootPath.parent != ".".toPath()) {
```

Comment:
@kgevorkyan please handle and check "." properly

Reply:
fixed